src/coreclr/src/jit/emitarm64.cpp

```diff
@@ -12082,8 +12091,40 @@ void emitter::emitDispIns(
                 assert(imm == 0);
                 if (id->idIsReloc())
                 {
-                    printf("RELOC ");
+                    printf("HIGH RELOC ");
                     emitDispImm((ssize_t)id->idAddr()->iiaAddr, false);
+                    size_t methodHandle = id->idDebugOnlyInfo()->idMemCookie;
+
+                    switch (methodHandle)
+                    {
+                        case GenTreeIntCon::MethodHandleType::Unknown:
+                            targetName = "Unknown";
+                            break;
+                        case GenTreeIntCon::MethodHandleType::StringLiteralNode:
+                            targetName = "Access_To_StringLiteralNode";
+                            break;
+                        case GenTreeIntCon::MethodHandleType::StaticLookupTree:
+                            targetName = "Static_LookupTree";
```

Comment:
I was not sure what is the right term to describe `StaticLookupTree` and `RuntimeLookupTree`. Any suggestions?

Reply:
Not needed anymore.